app/src/main/java/org/fossasia/susi/ai/rest/services/SusiService.java

```diff
@@ -84,4 +86,26 @@
      */
     @POST("/aaa/recoverpassword.json")
     Call<ForgotPasswordResponse> forgotPassword(@Query("forgotemail") String email);
+
+    /**
+     * User settings call
+     *
+      * @param key
+     * @param value
+     * @param accessToken
+     * @return the call
+     */
+    @POST("/aaa/changeUserSettings.json")
+    Call<ChangeSettingResponse> changeSettingResponse(@Query("key") String key,
+                                                      @Query("value") String value,
+                                                      @Query("access_token") String accessToken);
+
+    /**
+     * Get User settings
+     *
+     * @param accessToken
+     * @return settings
+     */
+    @GET("/aaa/listUserSettings.json")
+    Call<UserSetting> getUserSetting(@Query("access_token") String accessToken);
```

Comment:
end point api.susi.ai/aaa/listUserSettings and api.susi.ai/aaa/listUserSettings?access_token both work. I am using second one. no problem I will change it.